pyproject.toml

```diff
@@ -47,6 +47,7 @@ sphinx-rtd-theme = {version = "*", optional = true}
 numpydoc = {version = "*", optional = true}
 proposal = {version = "7.6.2", optional = true}
 pylfmap = {version = "*", optional = true}
+pygdsm = {version = "<=1.6.0", python = "<3.10", optional = true}
```

Comment:
does that mean only if you are on <=python3.9 you are using <=1.6?

Reply:
Yes